internal/exeutils/exeutils.go

```diff
@@ -21,17 +22,23 @@ func Executables(bins []string) ([]string, error) {
 	exes := []string{}
 
 	for _, bin := range bins {
-		err := filepath.Walk(bin, func(fpath string, info os.FileInfo, err error) error {
-			// Filter out files that are not executable
-			if info == nil || info.IsDir() || !fileutils.IsExecutable(fpath) { // check if executable by anyone
-				return nil // not executable
+		if fileutils.IsDir(bin) {
```

Comment:
This should always be the case, if anything if this isn't a dir we should return an error.